README.md

```diff
@@ -61,34 +61,42 @@ spec:
     scaleDownThresholdDurationSeconds: 1800
     scaleDownCooldownSeconds: 3600
     diskUsagePercentScaledownWatermark: 80
+  draining:
+    maxRetries: 999
+    maximumWaitTimeDurationSeconds: 30
+    minimumWaitTimeDurationSeconds: 10
```

Comment:
or: we expose it not through the EDS but through runtime-configuration (command-line properties) of the es-operator. I don't think people will want to configure this differently for each ES cluster they're running.

Reply:
That also works, I assumed it was intended per cluster since defined here.

Reply:
Hmm, I actually thought about adding it to the command line but then decided to go with Elasticsearch DataSet (EDS) configuration because, to me, it operates at the same abstraction level as scaling. If we control scaling behavior in the Elasticsearch DataSet (EDS) configuration, why don't we also control the draining behavior in the same place? Right now in the lounge, we have one cluster, but in the near future, we are migrating to new multi-tier clusters, and we might need different settings for scaling and draining. However, I'm not 100% sure if that will happen, or if we might end up having a single configuration for all of our clusters.

Reply:
To be honest, I can't see why some of the configurations (for example, priority-node-selectors) are defined in command-line properties. Putting configurations in EDS makes them easier to document, change (for instance, in case of an emergency, you can change it in Kubernetes without any need for deployment or even a restart), and more flexible. As I mentioned before, I believe the abstraction level is almost equivalent to scaling behavior, so it makes more sense to include it in the EDS.

Reply:
@mikkeloscar 
Regarding `experimental or `alpha`, the EDS will be sth like this?

```
experimental:
  draining:
    maxRetries: 999
    maximumWaitTimeDurationSeconds: 30
    minimumWaitTimeDurationSeconds: 10
```

Reply:
Done.